pkg/runctl/run_manager_test.go

```diff
@@ -1097,6 +1097,120 @@ func Test_RunManager_Start_CreatesTektonTaskRun(t *testing.T) {
 	assert.Assert(t, result != nil)
 }
 
+func Test_RunManager_addTektonTaskRunParamsForImage(t *testing.T) {
+	t.Parallel()
+	examinee := runManager{
+		pipelineRunsConfig: pipelineRunsConfigStruct{
+			Image:           "defaultImage1",
+			ImagePullPolicy: "defaultPolicy1",
+		},
+	}
+	for _, tc := range []struct {
+		name           string
+		spec           *stewardv1alpha1.PipelineSpec
+		expectedParams []tekton.Param
+	}{
+		{"empty",
+			&stewardv1alpha1.PipelineSpec{},
+			[]tekton.Param{
+				tektonStringParam("JFR_IMAGE", "defaultImage1"),
+				tektonStringParam("JFR_IMAGE_PULL_POLICY", "defaultPolicy1"),
+			},
+		}, {"no_image_no_policy",
+			&stewardv1alpha1.PipelineSpec{
+				JenkinsfileRunner: &stewardv1alpha1.JenkinsfileRunnerSpec{},
+			},
+			[]tekton.Param{
+				tektonStringParam("JFR_IMAGE", "defaultImage1"),
+				tektonStringParam("JFR_IMAGE_PULL_POLICY", "defaultPolicy1"),
+			},
+		}, {"image_only",
+			&stewardv1alpha1.PipelineSpec{
+				JenkinsfileRunner: &stewardv1alpha1.JenkinsfileRunnerSpec{
+					Image: "foo",
+				},
+			},
+			[]tekton.Param{
+				tektonStringParam("JFR_IMAGE", "foo"),
+				tektonStringParam("JFR_IMAGE_PULL_POLICY", "IfNotPresent"),
+			},
+		}, {"policy_only",
+			&stewardv1alpha1.PipelineSpec{
+				JenkinsfileRunner: &stewardv1alpha1.JenkinsfileRunnerSpec{
+					ImagePullPolicy: "bar",
+				},
+			},
+			[]tekton.Param{
+				tektonStringParam("JFR_IMAGE", "defaultImage1"),
+				tektonStringParam("JFR_IMAGE_PULL_POLICY", "bar"),
+			},
```

Comment:
This is wrong. If only the pull policy is specified, it must be ignored instead of being applied to the default image.

Reply:
I'm not sure